src/databaseInterface/src/main/kotlin/org/vaccineimpact/orderlyweb/db/AppConfig.kt

```diff
@@ -10,7 +10,7 @@ interface Config
 
 class AppConfig : Config
 {
-    private companion object
```

Comment:
This is the change that allows us to mutate the properties in the tests. I don't love this approach, but I also don't have a better solution at this point. The problem with passing a config object to the app `run` method and then registering it as an immutable singleton was that the tests are then also talking to that same singleton and can't mutate it (or access it for test setup before the Spark app has started and registered it.) I think a starting point would be to separate app config and test config into different classes, maybe I'll open a ticket to think about this further.